lte/gateway/python/integ_tests/defs.mk

```diff
@@ -128,7 +128,10 @@ s1aptests/test_attach_ul_udp_data_with_multiple_service_restart.py \
 s1aptests/test_attach_ul_udp_data_with_pipelined_restart.py \
 s1aptests/test_attach_ul_udp_data_with_sessiond_restart.py \
 s1aptests/test_service_req_ul_udp_data_with_mme_restart.py \
-s1aptests/test_attach_detach_setsessionrules_tcp_data.py
+s1aptests/test_attach_detach_setsessionrules_tcp_data.py \
+s1aptests/test_enable_ipv6_iface.py \
+s1aptests/test_ipv6_non_nat_dp_ul_tcp.py \
```

Comment:
I am seeing 3 test cases here.. Why are we adding only 1 test case in defs.mk file

Reply:
The DL test cases work fine individually but become flaky when added into integ-tests and the UL UDP testcase works but has out-of-order packets on trfserver side. Hence only one TC is added as of now. Once the others are fixed they will be added too.

Reply:
Makes sense. Can you please add the other test cases in commented mode somewhere downside in the defs.mk file with the comment mentioning this issue? This will help the test cases to be tracked.